mri/operators/fourier/cartesian.py

```diff
@@ -116,20 +116,20 @@ def op(self, img):
                 )
 
     def adj_op(self, x):
-        """ This method calculates inverse masked Fourier transform of a ND
+        """ This method computes the inverse masked Fourier transform of a ND
```

Comment:
(D401) First line should be in imperative mood. 
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute the inverse masked Fourier transform of a ND
```